locales/src/en/translation.json

```diff
@@ -221,10 +221,8 @@
         "instancenotfound": "Record not present. It may have expired.",
         "invalidediturl": "Not a valid edit URL",
         "loadfailed": "Failed to load __resource__",
-        "notfoundinformlist": [
-            "Oops, this form doesn't exist (anymore). Most likely the owner of the form has deleted it, archived it, or disabled it. Please contact the form's owner to confirm.",
-            "Detailed error: Form with ID __formId__ not found in /formList."
-        ],
+        "notfoundinformlist": "This form does not exist. Please check the URL for any missing characters.\n\nIf the form existed previously, it may have been archived, disabled or deleted. If this is unexpected, please contact the person who asked you to fill the form.\n\n(Attempted to access form with ID: __formId__)",
```

Comment:
Now that I see this in context, I wonder whether we should consider "This form does not exist" -> "This form does not exist or you no longer have access to it"? In many cases, a user no longer having access to a form will mean it's just not in the form list. I'm thinking something like that might be a useful hint for a project manager who might otherwise be very confused if the form is open and they can access it. On the other hand, that state might be a good enough indicator that it's permissions-related.

Another option would be "it may have been archived, disabled, deleted, or your access may have been revoked"

I don't feel strongly about it but did want to share the thought.

Reply:
And since you mentioned not leaking info about revoked resources in the PR description, @eyelidlessness, I do want to address that explicitly. I think adding it as a possibility in the list would be adequately non-revealing but I can also see not wanting to put that idea in an end user's head.